cm_sdk/api/__init__.py

```diff
@@ -0,0 +1,4 @@
+__author__ = 'e0d'
+
+class APIException(Exception):
```

Comment:
Seems like the appropriate place for package-wide definitions to me.  Suggest an alternative.